client/src/components/ChefSignUp.js

```diff
@@ -0,0 +1,155 @@
+import * as React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { UserContext } from "../contexts/user/UserContextProvider";
+import {
+    Box,
+    Button,
+    Checkbox,
+    Chip,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    Grid,
+    Input,
+    InputLabel,
+    ListItemText,
+    MenuItem,
+    Select,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+    chefButton: {
+        padding: "10px",
+    },
+    chips: {
+        display: "flex",
+        flexWrap: "wrap",
+    },
+    chip: {
+        margin: 2,
+        borderRadius: "10px",
+        background: theme.background.secondary,
+        color: "#ffffff",
+    },
+}));
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+const types = ["American", "Japanese", "Spanish"];
+
+const ChefSignUp = (props) => {
+    const user = React.useContext(UserContext);
+    const classes = useStyles();
+    const [open, setOpen] = React.useState(false);
+    const [specialty, setSpecialty] = React.useState([]);
+
+    const handleClickOpen = () => {
+        setOpen(true);
+    };
+
+    const handleClose = () => {
+        setOpen(false);
+    };
+    const handleChange = (event) => {
+        setSpecialty(event.target.value);
+    };
+
+    const updateUser = () => {
+        const formValues = {
+            isChef: true,
+            cuisineSpecialty: [...specialty],
```

Comment:
Should we add validation that there's at least one specialty?

Reply:
Yes, I can add that.  For timing sake I'll do that on my next PR so Gary can merge.